builder/builder.go

```diff
@@ -645,3 +644,22 @@ func (b *Builder) saveArtifactLocally(ctx context.Context, artifact domain.Artif
 	}
 	return nil
 }
+
+func (b *Builder) tempEarthlyOutDir() (string, error) {
+	tmpParentDir := filepath.Join(".", ".tmp-earthly-out")
```

Comment:
Is the filepath.Join needed, or can it just be this:
```suggestion
	tmpParentDir := ".tmp-earthly-out"
```

Reply:
You're right